interfaces/dummy-oauth/dummy-oauth.yaml

```diff
@@ -79,3 +63,498 @@ paths:
           description: >-
             The request was not properly formed
       summary: Generate an access token
+    post:
```

Comment:
That's right. Do you think /faa/uft/token in master is better, or we have this change in a faa-uft specific branch?

Reply:
The goal of the repository in general is to provide encapsulated functionality that can be leveraged appropriately in the future -- since the intent of this PR seems to be to provide the ability to easily obtain valid access tokens (what the dummy auth server does now), just using a different request/response API and the core capabilities, providing access to the core functionality via a different interface seems like something we would merge to master with the new interface available at a new endpoint.  (so, I'd imagine `/faa/uft/token` in master would be best)

The `interuss` repo generally generally tries to stay away from hosting prototyping branches as those can be hosted on appropriate forked repos.  Instead, we try to use the `interuss` repo to collect working functionality useful to most/all users of the repository with functionality only relevant to a subset of users encapsulated (separated) from the more common functionality.

Reply:
Changed to /fims/token. With removing fields from spec per the review it doesn't stay exactly like faa uft fims. But, very close to FIMS, close to what USSes have been using. Hence made the path - /fims/token. 
Changes can be made to spec as and when required. The current implementation worked while interfacing to uss_qualifier tests; keeping dummy auth implementation to provide the minimum required features required for POST token endpoint seems acceptable.